vcpkg/commands/common-options.md

```diff
@@ -29,6 +29,45 @@ Specifies the temporary path to store intermediate build files, such as objects
 
 Defaults to `buildtrees/` under the vcpkg root folder.
 
+### <a name="cmake-configure-debug"></a> `--x-cmake-configure-debug=<debugger-pipe>[;semicolon;port;list]`
+
+[!INCLUDE [experimental](../../includes/experimental.md)]
+
+Enables CMake's [`--debugger`](https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-debugger)
+within upstream build systems' `CMakeLists.txt`, such as within
+[`vcpkg_cmake_configure`](../maintainers/functions/vcpkg_cmake_configure.md).
+
+The `<debugger-pipe>` value is passed as
+[`--debugger-pipe`](https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-debugger-pipe) on the CMake
+command line.
+
+If there is a semicolon-separated port list, the debugger is only used for ports in that list. Otherwise, it is used
+for all ports.
+
+Note: The triplet variable `VCPKG_CMAKE_CONFIGURE_OPTIONS` is used to pass the options to the internal CMake configure call.
+As such it will not work for custom triplets overwriting `VCPKG_CMAKE_CONFIGURE_OPTIONS`.
+Either manually pass the debugging options or append your options to `VCPKG_CMAKE_CONFIGURE_OPTIONS` instead.
+
+Note: Unlike `--x-cmake-debug` there will be no output indicating that the build is waiting for a debugger to connect. The output will simply stall at `Configuring <triplet>`.
+
+Note: Although all ports should be calling CMake inside of the portfile to obtain compile flags from the CMake toolchain. This command is mainly useful for ports actually using CMake as their build system.
+
+### <a name="cmake-debug"></a> `--x-cmake-debug=<debugger-pipe>[;semicolon;port;list]`
+
+[!INCLUDE [experimental](../../includes/experimental.md)]
+
+Enables CMake's [`--debugger`](https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-debugger) while
+running directly-invoked CMake scripts like triplet `.cmake`s or `portfile.cmake`s.
```

Comment:
```suggestion
running directly-invoked CMake scripts like triplet files (for example, `x64-windows.cmake`) or `portfile.cmake` files.
```

> `.cmake`s or `portfile.cmake`s 

Mixing code-style with regular font looks strange when rendered in the Learn platform. I'm also concerned that it won't localize cleanly and might cause issues with screen readers.